integration-test/src/main/java/com/datastrato/gravitino/integration/test/MiniGravitino.java

```diff
@@ -134,7 +134,12 @@ public void start() throws Exception {
       if (started || future.isDone()) {
         break;
       }
-      Thread.sleep(500);
+      try {
```

Comment:
Prefer using 
`https://guava.dev/releases/19.0/api/docs/com/google/common/util/concurrent/Uninterruptibles.html#sleepUninterruptibly(long,%20java.util.concurrent.TimeUnit)`
to simplify the code of this class.

Reply:
The 'sleepUninterruptibly' method only call 'Thread.interrupt()' to keep the state of interrupted, but will not re-throw the exception which is necessary sometimes. 

so i will only use 'sleepUninterruptibly' to instead of where re-throw exception is not necessary, ok ? @qqqttt123

Reply:
Yes, it's not necessary to re-throw exception actually. This class is mainly used to test.